oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -1423,7 +1426,8 @@ void pldm::responder::oem_ibm_platform::Handler::handleBootTypesAtPowerOn()
     BiosAttributeList biosAttrList;
     auto bootInitiator = getBiosAttrValue("pvm_boot_initiator_current");
     std::string restartCause;
-    if (bootInitiator != "HMC" && !bootInitiator.empty())
+    if (((bootInitiator != "HMC") || (bootInitiator != "Host")) &&
```

Comment:
Did you get a chance to confirm that when we boot with "Host" that PHYP resets it once they are booted? We need to be sure "someone" clears it or it will never be cleared.

Reply:
yes, I did confirm that on Poweron PHYP resets these 2 bios attributes to default values.